circle.yml

```diff
@@ -15,6 +15,10 @@ test:
           fi
       : parallel: true
   override:
+    - bin/run-integration :
+        parallel: true
+        files:
+          - integration/**/*.js
```

Comment:
I forgot that we already use glob with `TEST_ROOT=integration` configured in `npm run test-integration`  :)

We can skip files part here and it should still work exactly the same.